src/verifiers/documentStatus/didSigned/didSignedDocumentStatus.ts

```diff
@@ -79,6 +79,7 @@ const verifyV2 = async (
       case v2.RevocationType.None:
         return Promise.resolve({ revoked: false });
       default:
+        // TODO unreachable
```

Comment:
is this unreachable according to static verification or its actually runtime unreachable?

Reply:
runtime unreachable